study-builder/studies/brain/release.conf

```diff
@@ -276,6 +276,21 @@
       "blocks": [
         {
           "componentType": "MAILING_ADDRESS",
+          "titleTemplate": {
```

Comment:
just to keep UpdateTemplatesInPlace happy because it compares titleTemplates from conf.